src/app/pages/client/suggestions/suggestions.page.spec.ts

```diff
@@ -0,0 +1,24 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { IonicModule } from '@ionic/angular';
+
+import { SuggestionsPage } from './suggestions.page';
+
+describe('SuggestionsPage', () => {
+  let component: SuggestionsPage;
+  let fixture: ComponentFixture<SuggestionsPage>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [ SuggestionsPage ],
+      imports: [IonicModule.forRoot()]
+    }).compileComponents();
+
+    fixture = TestBed.createComponent(SuggestionsPage);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  }));
+
+/*  it('should create', () => {
+    expect(component).toBeTruthy();
+  });*/
```

Comment:
Remove unused code. Ignore this comment if this the part in need of server functions implementation.

Reply:
Removed it